site2/docs/reference-configuration-broker.md

```diff
@@ -0,0 +1,3879 @@
+# Broker
+## Required
+### clusterName
+Name of the cluster to which this broker belongs to
+
+**Default**: `null`
+
+**Dynamic**: `false`
+
+**Category**: Server
+
+## Optional
+### acknowledgmentAtBatchIndexLevelEnabled
+Whether to enable the acknowledge of batch local index
+
```

Comment:
@gaoran10 is this correct?
<img width="569" alt="image" src="https://user-images.githubusercontent.com/50226895/180017827-95e7a9df-2d13-4180-b5e4-ba89e2e00759.png">

Reply:
I think it's ok.